app/views/items/new.erb

```diff
@@ -1,9 +1,7 @@
 <form action="/items" method="post">
   <p>Enter a new item:</p>
-  <input type='text' name='item[name]'/><br/>
-  <input type='integer' name='item[id]'/><br/>
-  <input type='description' name='item[description]'/><br/>
-  <input type='decimal' name='item[price]'/><br/>
-  <input type='text' name='item[image]'/><br/>
+  <input type='text' name='item[name]' value= "Name"/><br/>
+  <input type='description' name='item[description]' value= "Description"/><br/>
+  <input type='decimal' name='item[price]' value= "Price"/><br/>
```

Comment:
So for this, I found a really nice way to make those boxes explicit. 

by doing 
<input type='text' name='item[name]' value= "Name" placeholder="Enter new name here"/><br/>

we get that text with the placeholder element inside the box, when the user clicks on the box, that goes away. its like the google search bar or something.

Reply:
dig it. let's implement it in the next pull request.